frontend/src/components/Election/Voting/VotePage.tsx

```diff
@@ -45,35 +57,26 @@ const VotePage = () => {
     let pages = election.races.map((race, i) => {
       let candidates = race.candidates.map(c=> ({...c, score: null}))
       return {
-        type: "ballot",
+        instructionsRead: false,
         candidates: election.settings.random_candidate_order ? shuffle(candidates) : candidates,
         voting_method : race.voting_method,
         race_index: i
-    }
-  })
-
-    // determine where to add info pages
-    // commented out for now in case we want to return to this
-    // for(var i = 0; i < pages.length; i++){
-    //   if(pages[i].type != "ballot") continue;
-
-    //   // check if page is the first race with the voting method
-    //   var info_exists = pages.some((p) => p.type == 'info' && p.voting_method == pages[i].voting_method);
-    //   if(info_exists) continue;
-      
-    //   // add info page for method
-    //   pages.splice(i, 0, {
-    //     type: "info",
-    //     voting_method: pages[i].voting_method
-    //   })
-    // }
-
+      }
+    })
     return pages
   }
+
   const { id } = useParams();
   const [pages, setPages] = useState(makePages())
   const navigate = useNavigate();
   const [currentPage, setCurrentPage] = useState(0)
+
+  const setInstructionsRead = () => {
+    pages[currentPage].instructionsRead = true;
```

Comment:
How does this work? pages should be immutable as a state

Reply:
Yeah, that operation a lone won't trigger a refresh so that's why I call setPages

It's a bit hacky, but the alternative is to have a separate useState where I represent instructionsRead as an array and I'd need to do the same trick anyway